easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'ConfigureMake'
+
+name = 'coevol'
+commit_id = '5d7fc52'
+version = '%s-02012018' % commit_id
```

Comment:
@pescobar Wouldn't it be better to simply use `20180201` as `version` (and don't mention the commit ID in the `version`), since that sorts better?

Reply:
@pescobar please see #6642